src/parser/demonhunter/vengeance/modules/spells/azeritetraits/RevelInPain.js

```diff
@@ -0,0 +1,93 @@
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import SPELLS from 'common/SPELLS';
+import Events from 'parser/core/Events';
+import AzeritePowerStatistic from 'interface/statistics/AzeritePowerStatistic';
+import { formatNumber, formatPercentage } from 'common/format';
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
+import React from 'react';
+
+/**
+ * Infernal Armor Azerite Power
```

Comment:
looks like the description is right but title in the comment is wrong?

Reply:
Copy/Paste fails . . . fixed